core/trino-main/src/main/java/io/trino/sql/planner/SubqueryPlanner.java

```diff
@@ -117,7 +117,7 @@ public PlanBuilder handleSubqueries(PlanBuilder builder, Expression expression,
         for (Cluster<InPredicate> cluster : cluster(builder.getScope(), selectSubqueries(builder, expression, subqueries.getInPredicatesSubqueries()))) {
             builder = planInPredicate(builder, cluster, subqueries);
         }
-        for (Cluster<SubqueryExpression> cluster : cluster(builder.getScope(), selectSubqueries(builder, expression, subqueries.getScalarSubqueries()))) {
+        for (Cluster<SubqueryExpression> cluster : cluster(builder.getScope(), selectSubqueries(builder, expression, subqueries.getSubqueries()))) {
```

Comment:
The rename in this and other classes could be extracted to another commit.
In fact, it logically belongs to this PR: https://github.com/trinodb/trino/pull/7774

Reply:
Pulled it out into a separate commit.